package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-spinner-button",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
react-native-spinner-button in latest version 1.0.3 instead of 1.0.4. so keep it as it is.

Reply:
changes are done.